File: Team1/RiyaSingh/moveZeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int n = 0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=0){
                nums[n++]=nums[i];
            }
        }
        while(n<nums.length){
            nums[n++]=0;
        }
    }
}

File: Team2/Riya_Gupta/MoveZeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        
        int k = 0;
        
        for(int i = 0 ; i < nums.size() ; i++)
        {
            if(nums[i]!=0)
            {
                nums[k] = nums[i]; // move non zeroes ahead
                
                if(k!=i)
                nums[i]=0;
                
                 k++;
            }
        }
 
    }
};
